lib/transformer.dart

```diff
@@ -474,12 +475,12 @@ class InitializerData {
 }
 
 // Reads a file list from a barback settings configuration field.
-_readFileList(BarbackSettings settings, String field) {
+List<String> _readFileList(BarbackSettings settings, String field) {
   var value = settings.configuration[field];
   if (value == null) return null;
-  var files = [];
+  List<String> files = [];
   bool error;
-  if (value is List) {
+  if (value is List<String>) {
```

Comment:
This really doesn't do what you think, at least without strong mode. In regular mode if value is a `List<dynamic>` (ie: pretty much every list), then this will return true. This is why it does the explicit check below even though it feels redundant.